pnpm-workspace.yaml

```diff
@@ -0,0 +1,4 @@
+packages:
+ - 'packages/*'
+ - 'packages/@ember/*'
+ - 'packages/@glimmer/*'
```

Comment:
When I converted this repo to pnpm, it looks like pnpm was auto-detecting all our packages. But it turns out it was not detecting all of them. This makes it explicit.